package.json

```diff
@@ -67,6 +67,7 @@
     "@restart/context": "^2.1.4",
     "@restart/hooks": "^0.3.11",
     "@types/react": "^16.8.23",
+    "bs-custom-file-input": "^1.3.4",
```

Comment:
Well without it the input will not even say what file has been added/uploaded. It will be stuck saying "select a file" or whatever label you have defined. I think as a default behavior it is lacking maybe to much without the script included?

Reply:
hmm I agree that the experience isn't great but adding a third party library we don't know anything about isn't good either, especially for optional behavior, anyone using any Form field is going to end up with this in their bundle, even tho it's small.

The expectation wouldn't be to include this since upstream doesn't. I don't see a lot wrong with doing the same thing as them and including it the docs but not the bundle

@taion @bpas247 what do you think?

Reply:
With your arguments above I'm going to remove it and add it to the docs as bootstrap does. Thank you for the input! :)

Reply:
The script has been removed and I updated the docs. I do not add the script into the www docs either. Instead I added some text that should guide whomever reads it towards achieving it with the plugin.